lib/src/play_store_search_api.dart

```diff
@@ -138,11 +138,42 @@ class PlayStoreResults {
   static String? version(Document response) {
     String? version;
     try {
-      final additionalInfoElements = response.getElementsByClassName('hAyfc');
-      final versionElement = additionalInfoElements.firstWhere(
-        (elm) => elm.querySelector('.BgcNfc')!.text == 'Current Version',
-      );
-      final storeVersion = versionElement.querySelector('.htlgb')!.text;
+      const patternName = ",\"name\":\"";
```

Comment:
@humanolaranja Thanks for the code changes in this PR. I would like to maintain support for the old Play Store format, which will also keep the existing tests passing. Can you call this new code in a new method only when the current code fails? Then, you could add a new test_play_store_page for testing.

Reply:
Sure, I'll update it